gapic-generator-java/src/main/java/com/google/api/generator/gapic/protowriter/Writer.java

```diff
@@ -88,6 +85,23 @@ public static CodeGeneratorResponse write(
     return response.build();
   }
 
+  public static CodeGeneratorResponse write(
```

Comment:
Where is this method called?

Reply:
This is the production method used by `Generator`:

https://github.com/googleapis/sdk-platform-java/blob/56775a61a054fb67a15023481cd3d0a0cf2b0496/gapic-generator-java/src/main/java/com/google/api/generator/gapic/Generator.java#L35

The one I created is for testing purposes, where we can provide our own output stream.

Reply:
I see, thanks.